### PR TITLE
feat(query-preview): modify translations and styles

### PR DESCRIPTION
**Pull request by tiborux**
EX-6837

### CODE REVIEW

File: src/i18n/messages/en.messages.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
       "message": "Those who searched for <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> also viewed:"
     },
     "nextQueryPreview": {
-      "title": "Next Queries",
+      "message": "Those who searched for <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> also viewed:",
```
Comment (javieri-empathy) [inline src/i18n/messages/en.messages.json:41]: There are no `"` in the designs for this message's query.

File: src/components/desktop/desktop-next-query-preview.vue [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
   <NextQueryPreview
     #default="{ results, totalResults, suggestion }"
-    class="x-list x-background--neutral-95 x-list--gap-06 x-padding--06"
+    class="x-list x-background--neutral-95 x-list--gap-05 x-padding--06"
     :suggestion="nextQuery"
   >
-    <span class="x-title2 x-font-weight--bold">{{ $t('nextQueryPreview.title') }}</span>
-    <div class="x-list x-list--horizontal x-list--align-center x-list--gap-03">
+    <span v-html="$t('nextQueryPreview.message', { query: $x.query.search })" class="x-title3" />
+    <div class="x-list x-list--horizontal x-list--align-center x-list--gasp-03">
```
Comment (javieri-empathy) [inline src/components/desktop/desktop-next-query-preview.vue:8]: Typo in `gasp` -> `gap`

File: src/components/mobile/mobile-next-query-preview.vue [hunk 3]
```diff
@@ -31,9 +32,10 @@
...
       <NextQuery
         :suggestion="nextQuery"
-        class="x-button x-button--ghost x-padding--00 x-margin--left-auto x-font-weight--bold"
+        class="x-margin--left-auto x-list x-list--gap-04 x-font-weight--bold"
       >
         {{ $t('nextQueryPreview.viewResults') }}
+        <ChevronRightIcon />
       </NextQuery>
     </div>
     <SlidingPanel
```
Comment (javieri-empathy) [inline src/components/mobile/mobile-next-query-preview.vue:41]: Probably not from this task, but there is a vertical scroll in the sliding panel

File: src/i18n/messages/en.messages.json (hunk 1 above)
Comment (javieri-empathy) [reply src/i18n/messages/en.messages.json:41]: Mobile messages are different for this property, but I think there is a typo. Can we confirm that we want to leave them like this?

**Review by javieri-empathy — CHANGES_REQUESTED** [review]

File: src/i18n/messages/en.messages.json (hunk 1 above)
Comment (tiborux) [reply src/i18n/messages/en.messages.json:41]: Yes, if you check the task, it said that both messages have to be the same

File: src/components/desktop/desktop-next-query-preview.vue [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <template>
   <NextQueryPreview
     #default="{ results, totalResults, suggestion }"
-    class="x-list x-background--neutral-95 x-list--gap-06 x-padding--06"
+    class="x-list x-background--neutral-95 x-list--gap-05 x-padding--06"
     :suggestion="nextQuery"
   >
-    <span class="x-title2 x-font-weight--bold">{{ $t('nextQueryPreview.title') }}</span>
-    <div class="x-list x-list--horizontal x-list--align-center x-list--gap-03">
+    <span v-html="$t('nextQueryPreview.message', { query: $x.query.search })" class="x-title3" />
```
[the remark this answers is not in the document]
Comment (tiborux) [reply src/components/desktop/desktop-next-query-preview.vue:7]: uhhh nice, i like it

**Review by mavmaf — COMMENTED** [review]
Hey!
After checking with Cris, please:
- _Change the gap between result-cards and button to spacing-06 (Desktop)_ This was already in the task description but it seems it's set to 05
- Add spacing-02 between NQ name and (number of results)

**Comment by tiborux** [comment]
> Hey! After checking with Cris, please:
> 
> * _Change the gap between result-cards and button to spacing-06 (Desktop)_ This was already in the task description but it seems it's set to 05
> * Add spacing-02 between NQ name and (number of results)

We cannot change the gap between the results-card and the button, because the gap is in the wrapper component, so If we wanna achieve that, we have to set to gap-05 and add a margin-top-03 in the button to simulate that it has a gap-06

File: src/i18n/messages/es.messages.json [hunk 5]
```diff
@@ -37,10 +37,10 @@
       "message": "Los que buscaron <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> también han visto:"
     },
     "nextQueryPreview": {
-      "title": "Next Queries",
+      "message": "Los que buscaron <span class=\"x-text x-text--bold\">\"{query}\"</span> también han visto:",
```
Comment (javieri-empathy) [inline src/i18n/messages/es.messages.json:40]: This message still has HTML

**Review by mavmaf — DISMISSED** [review]

File: src/i18n/messages/en.messages.json [hunk 6]
```diff
@@ -103,10 +103,10 @@
   },
   "mobile": {
     "nextQueriesGroup": {
-      "message": "Those who searched for <span class=\"x-text x-text--bold\">\"{query}\"</span> also viewed:"
+      "message": "Those who searched for <span class=\"x-text x-text--bold x-font-size--05\">\"{query}\"</span> also viewed:"
```
Comment (javieri-empathy) [inline src/i18n/messages/en.messages.json:106]: Didn't you roll back this change?

**Review by javieri-empathy — APPROVED** [review]

**Review by mavmaf — APPROVED** [review]

**Pull request merged**